### PR TITLE
[KHRGA_30] Updated command accessor

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA_30

### CODE REVIEW

File: source/iface/command-accessor.rst [hunk 1]
```diff
@@ -6,281 +6,290 @@
...
-  atomic<dataT, sycl::access::address_space::global_space> operator[](
-    size_t index) const;
-  atomic<dataT, sycl::access::address_space::global_space> operator[](
-    id<dimensions> index) const;
+.. _tags_buff_accessors:
 
-  *Single dimension in multi-dimensional buffer*
-  __unspecified__ &operator[](size_t index) const;
+=============================
+Buffer command accessors tags
```
Comment (mkinsner) [inline source/iface/command-accessor.rst:221]:
```suggestion
Buffer accessor tags
```

File: source/iface/command-accessor.rst [hunk 2]
```diff
@@ -6,281 +6,290 @@
...
+
+::
+
+  template <access::decorated IsDecorated> accessor_ptr
+
+If ``(AccessTarget == target::device):
+multi_ptr<value_type, access::address_space::global_space, IsDecorated>.``
+
+The definition of this type is not specified when
+``(AccessTarget == target::host_task)``.
```
Comment (mkinsner) [inline source/iface/command-accessor.rst:59]: Please move this section until after the constructors and after the "member functions" sections.  It's less important than those other sections and may be confusing if described first

File: source/iface/command-accessor.rst [hunk 3]
```diff
@@ -6,281 +6,290 @@
...
-``dimensions``   number of dimensions
-===============  ===
+``get_pointer``
+===============
 
+::
 
-Returns coordinates of the origin of the buffer or offset that was
-provided when the accessor was created.
+  global_ptr<access::decorated::legacy> get_pointer() const noexcept
```
Comment (mkinsner) [inline source/iface/command-accessor.rst:180]: Is the other form missing from this list?
`std::add_pointer_t<value_type> get_pointer() const noexcept`

File: source/iface/command-accessor.rst (hunk 1 above)
Comment (imvucic) [reply source/iface/command-accessor.rst]: Fixed with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/94/commits/0c470ac494e971cde631e7ae798de38f0b11bdac

File: source/iface/command-accessor.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/command-accessor.rst]: Removed with https://github.com/KhronosGroup/SYCL_Reference/pull/94/commits/0c470ac494e971cde631e7ae798de38f0b11bdac

File: source/iface/command-accessor.rst (hunk 3 above)
Comment (imvucic) [reply source/iface/command-accessor.rst:163]: Added with https://github.com/KhronosGroup/SYCL_Reference/pull/94/commits/0c470ac494e971cde631e7ae798de38f0b11bdac

Comment (mkinsner) [reply source/iface/command-accessor.rst:163]: Thanks.  With the addition as it is now, it's not clear what API is deprecated (there is one line between this API and the previous one, saying "deprecated in 2020. ..."

Can this please be clarified?  It needs to be clear which of the APIs is/are deprecated.

Comment (imvucic) [reply source/iface/command-accessor.rst:163]: Updated with commit https://github.com/KhronosGroup/SYCL_Reference/pull/94/commits/d94161d3a43c761647f2c1926137140abcf42255

Comment (mkinsner) [reply source/iface/command-accessor.rst:163]: Which form of `get_pointer` the comment refers to still isn't clear.  "This version" could refer to the version of `get_pointer` either on the line preceding or following.  "The version" could be changed to either "The previous version" or "The following version" to clarify.

image

Comment (imvucic) [reply source/iface/command-accessor.rst:163]: I totally agree...added "The previous version" with commit : https://github.com/KhronosGroup/SYCL_Reference/pull/94/commits/aa2f21f4449fd57c51843413d927c72559bbf108

**Pull request merged**